ledger/src/blockstore_processor.rs

```diff
@@ -582,6 +586,8 @@ fn process_entries_with_callback(
                 }
             }
             EntryType::Transactions(transactions) => {
+                prioritization_fee_cache.update_transactions(bank.slot(), transactions.iter());
```

Comment:
This will be taking the same lock on the cache as the RPC serving requests right? I think this call should be offloaded to the fee_cache thread. We can maybe do this without cloning the transactions by wrapping the `TransactionBatch` created below in an `Arc` and then passing that over to the fee_cache thread via a `FinalizingSerivceUpdate::TransactionUpdate(slot, Arc<TransactionBatch>)`

Reply:
TransactionBatch has lifetimes, makes Arc-wrap then pushing through channel difficult. While working on this, realized that if SanitizedTransaction's "writable accounts" are cached and can be reused, not only it makes pushing entire min-fee update to second thread possible (this comment), but will also yield opportunities to improve overall performance. PR #27263 is an attempt.

Reply:
this is addressed in https://github.com/taozhu-chicago/solana/pull/8